detectron2/utils/visualizer.py

```diff
@@ -463,7 +463,18 @@ def draw_panoptic_seg_predictions(
 
         return self.output
 
-    def draw_dataset_dict(self, dic):
+    def draw_dataset_dict(self, dic, ignore_label=255):
```

Comment:
It's not a good idea to add an argument that's only specific to semantic segmentation here.
A reasonable fix would be to modify `draw_sem_seg` so that it does not draw anything that does not have a class name. I think that's already done in this PR as well so the change in this method seems unnecessary.

A even better fix is to include ignored labels as the metadata for semantic segmentation datasets.

Reply:
I was thinking that IGNORE_VALUE can be one of the registered classes. But if this corner case is not something the codebase considers, yeah, filtering by the class name is enough. Let me know how you want to proceed.

I like the idea with metadata. Will you come with a commit to fix this? I am not sure where the best place is to pass  IGNORE_VALUE from cfg to metadata. One place I can think about is in the main function of `visualize_data.py`, but it also looks too specific.

Reply:
I think only keeping the filtering is the best solution for now. Metadata is attribute of the dataset and should be set when registering the dataset. This can be done in later commits.

Reply:
Cool, I will proceed with this.